src/content/blog/2024-08-05-rafiki-code-architecture.mdx

```diff
@@ -0,0 +1,203 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "Architecture Breakdown of Rafiki (Pending Title)"
+description: 
+date: 2024-08-05
+slug: rafiki-low-level-intro
+author: Nathan Lie
+author_url: https://www.linkedin.com/in/nathan-lie-138a73121
+tags:
+  - Interledger
```

Comment:
```suggestion

```

We removed Blair's Interledger tag too, since we are talking about Interledger engineering blog it seems redundant.